lib/pages/member/member_management/member_management_widget.dart

```diff
@@ -515,3 +499,101 @@ Widget modalSheet(
     ],
   );
 }
+
+leaveChannel(Channel currentChannel, Account user, BuildContext context) async {
+  showDialog<String>(
+    context: context,
+    builder: (BuildContext context) => AlertDialog(
+      backgroundColor: Get.theme.colorScheme.secondaryContainer,
+      shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.all(Radius.circular(18.0))),
+      title: Text(
+        '${AppLocalizations.of(context)!.leaveChannel}  ${currentChannel.name}',
+        style: Theme.of(context).textTheme.headline1!.copyWith(fontSize: 16),
+      ),
+      content: Text(
+        AppLocalizations.of(context)!.leaveChannelWarning,
+        style: Theme.of(context)
+            .textTheme
+            .headline1!
+            .copyWith(fontSize: 15, fontWeight: FontWeight.normal),
+      ),
+      actions: <Widget>[
+        SizedBox(
+          height: 45,
+          width: 80,
+          child: ButtonTextBuilder(
+            Key('button_2'),
+            onButtonClick: () => Navigator.pop(context),
+            backgroundColor: Get.isDarkMode
+                ? Get.theme.backgroundColor
+                : Get.theme.colorScheme.secondary,
+          )
+              .setWidth(double.infinity)
+              .setHeight(50)
+              .setText(AppLocalizations.of(context)!.cancel)
+              .setTextStyle(Get.isDarkMode
+                  ? Get.theme.textTheme.headline1!
+                      .copyWith(fontSize: 17, fontWeight: FontWeight.w600)
+                  : Get.theme.textTheme.bodyText1!
+                      .copyWith(fontSize: 17, fontWeight: FontWeight.w500))
+              .build(),
+        ),
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: SizedBox(
+            height: 45,
+            width: 80,
+            child: ButtonTextBuilder(
+              Key('button_1'),
```

Comment:
`button_1` has no meaning. Consider changing it into something else that is more meaningful

Reply:
fix